### PR TITLE
setup.py.in: Modify quotes

### PR DESCRIPTION
**Pull request by Adrianzatreanu**

### CODE REVIEW

File: bears/java/InferBear.py [hunk 1]
```diff
@@ -9,13 +9,24 @@ class InferBear:
...
     LICENSE = 'AGPL-3.0'
     ASCIINEMA_URL = 'https://asciinema.org/a/1g2k0la7xo5az9t8f1v5zy66q'
     CAN_DETECT = {'Security'}
 
-    @staticmethod
-    def create_arguments(filename, file, config_file):
-        return '-npb', '--', 'javac', filename
+    def create_arguments(self, filename, file, config_file, language: str):
+        """
+        :param language: The language to analyze (Java, Objective-C or C).
```
Comment (adtac) [inline bears/java/InferBear.py]: what about the other args?

Edit: only `config_file`

File: bears/java/InferBear.py [hunk 2]
```diff
@@ -9,13 +9,24 @@ class InferBear:
...
+        """
+        language_args = {
+            'java': ('javac',),
+            'objective-c': ('clang', '-c'),
+            'c': ('gcc', '-c')
+        }
+        if language not in language_args:
```
Comment (adtac) [inline bears/java/InferBear.py]: `language.lower()`

File: tests/java/InferBearTest.py [hunk 3]
```diff
@@ -20,8 +22,66 @@ class InferBad {
...
+    tempfile_kwargs={"suffix": ".java"}, settings={'language': 'bullshit'})
+
 
-InferBearTest = verify_local_bear(InferBear,
-                                  valid_files=(good_file,),
-                                  invalid_files=(bad_file,),
-                                  tempfile_kwargs={"suffix": ".java"})
+InferBearObjCTest = verify_local_bear(
+    InferBear, valid_files=(good_objc_file,), invalid_files=(bad_objc_file,),
+    tempfile_kwargs={"suffix": ".m"}, settings={'language': 'objective-c'})
```
Comment (adtac) [inline tests/java/InferBearTest.py]: where is the `gcc` C test? :)

Comment (Adrianzatreanu) [inline tests/java/InferBearTest.py]: its a wip ;p marked it now

**Comment by Adrianzatreanu** [comment]
seems i cannot install infer on my macOS and i cannot reproduce it locally, have no idea why it fails, HELP WANTED THX :D 

CC @jayvdb

File: tests/java/InferBearTest.py (hunk 3 above)
Comment (adtac) [inline tests/java/InferBearTest.py]: You can enable SSH on one of the Circle builds and use that as your linux machine.

**Comment by AbdealiJK** [comment]
@Adrianzatreanu Why does it not work on your Mac ?
```
$ brew install infer
==> Downloading https://homebrew.bintray.com/bottles/infer-0.9.2.el_capitan.bottle.tar.gz
######################################################################## 100.0%
==> Pouring infer-0.9.2.el_capitan.bottle.tar.gz
```

**Comment by AbdealiJK** [comment]
Ah, I notice that the default python in Mac is not nice. And gives a `ValueError: unknown locale: UTF-8` ...
To fix this (for ssh and python .. it's a mac issue), follow https://coderwall.com/p/-k_93g/mac-os-x-valueerror-unknown-locale-utf-8-in-python and just put:
```
export LC_ALL=en_US.UTF-8
export LANG=en_US.UTF-8
```
in ~/.bashrc or ~/.profile

**Comment by Adrianzatreanu** [comment]
@AbdealiJK

(venv) dhcp-138-246-42-17:coala-bears adrianzatreanu$ brew install infer
Warning: You are using OS X 10.12.
We do not provide support for this pre-release version.
You may encounter build failures or other breakages.
Please create pull-requests instead of filing issues.
==> Installing dependencies for infer: gringo, clasp, aspcud, opam
==> Installing infer dependency: gringo
==> Downloading https://downloads.sourceforge.net/project/potassco/gringo/4.5.4/gringo-
Already downloaded: /Users/adrianzatreanu/Library/Caches/Homebrew/gringo-4.5.4.tar.gz
==> /usr/local/opt/scons/bin/scons --build-dir=release gringo clingo reify
Last 15 lines from /Users/adrianzatreanu/Library/Logs/Homebrew/gringo/01.scons:
2016-10-31 09:29:50 +0100

/usr/local/opt/scons/bin/scons
--build-dir=release
gringo
clingo
reify

scons: *** SCons version 2.5.0 does not run under Python version 3.5.2.
Python 3 is not yet supported.

READ THIS: https://git.io/brew-troubleshooting
If reporting this issue please do so at (not Homebrew/brew):
  https://github.com/Homebrew/homebrew-core/issues

Warning: You are using OS X 10.12.
We do not provide support for this pre-release version.
You may encounter build failures or other breakages.
Please create pull-requests instead of filing issues.

**Comment by AbdealiJK** [comment]
@Adrianzatreanu as you know, infer only works with python 2 ... disable your virtualenv which has python 3.
A `brew doctor` will warn you about your symbolic links pointing to python3 also which confuses brew.

**Comment by Adrianzatreanu** [comment]
Adrians-MacBook-Pro:~ adrianzatreanu$ brew install infer
Warning: You are using OS X 10.12.
We do not provide support for this pre-release version.
You may encounter build failures or other breakages.
Please create pull-requests instead of filing issues.
==> Installing dependencies for infer: aspcud, opam
==> Installing infer dependency: aspcud
==> Downloading https://downloads.sourceforge.net/project/potassco/aspcud/1.9.1/
Already downloaded: /Users/adrianzatreanu/Library/Caches/Homebrew/aspcud-1.9.1.tar.gz
==> cmake .. -DCMAKE_C_FLAGS_RELEASE=-DNDEBUG -DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG
Last 15 lines from /Users/adrianzatreanu/Library/Logs/Homebrew/aspcud/01.cmake:
CMake Error at /usr/local/Cellar/cmake/3.6.1/share/cmake/Modules/Platform/Darwin.cmake:76 (message):
  CMAKE_OSX_DEPLOYMENT_TARGET is '10.12' but CMAKE_OSX_SYSROOT:

   ""

  is not set to a MacOSX SDK with a recognized version.  Either set
  CMAKE_OSX_SYSROOT to a valid SDK or set CMAKE_OSX_DEPLOYMENT_TARGET to
  empty.
Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.6.1/share/cmake/Modules/CMakeSystemSpecificInformation.cmake:36 (include)
  CMakeLists.txt:9 (project)


-- Configuring incomplete, errors occurred!
See also "/tmp/aspcud-20161031-48638-9lt7c9/aspcud-1.9.1-source/build/CMakeFiles/CMakeOutput.log".

READ THIS: https://git.io/brew-troubleshooting
If reporting this issue please do so at (not Homebrew/brew):
  https://github.com/Homebrew/homebrew-core/issues

Warning: You are using OS X 10.12.
We do not provide support for this pre-release version.
You may encounter build failures or other breakages.
Please create pull-requests instead of filing issues.

**Comment by Mixih** [comment]
@Adrianzatreanu Hello, Just checking in :). Did you figure out the problems?

**Comment by sils** [comment]
ack https://github.com/coala/coala-bears/pull/962/commits/79ce3c5c5a9eebf7cfbec5e18fded80d648ecbca

**Comment by sils** [comment]
@rultor merge

**Pull request merged**